ui-participant/src/hub/documents/DocumentLibrary.tsx

```diff
@@ -20,10 +27,20 @@ export default function DocumentLibrary() {
     pStudy.study.studyEnvironments.find(studyEnv =>
       studyEnv.environmentName === portalEnv.environmentName))?.study
 
```

Comment:
I don't think this is handling multi-study correctly. We're routing this outside of a particular study, so there isn't a current study. (e.g. the path is just 'documents', not 'study/studyName/documents'. So we need to either route this to a specific study (by having a study selector), or be explicit that this is including all documents across the portal (which I think is correct)